### PR TITLE
AM-3134 Create new role for SSCS post-hearing-judge 

### PR DESCRIPTION
**Pull request by mikebrownccd**
https://tools.hmcts.net/jira/browse/AM-3134
Create new role for SSCS post-hearing-salaried-judge

> **NB: This PR is Dependent  on RAS hmcts/am-role-assignment-service#2159.**
> ~**NB: This PR has temporary changes to Jenkins and Chart file configuration that need reversing before merger.**~

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] Does this PR introduce a breaking change

### CODE REVIEW

**Review by prashanthkumarnb — APPROVED** [review]

File: Jenkinsfile_CNP
Comment (mattnayler) [inline Jenkinsfile_CNP:79]: NB: This temporary change must be reverted before merging.

**Review by mattnayler — APPROVED** [review]
LGTM

**Review by mattnayler — APPROVED** [review]
Re-approved now that FTAs are green.

File: Jenkinsfile_CNP
Comment (mattnayler) [inline Jenkinsfile_CNP:79]: NB: This temporary change must be reverted before merging.

File: charts/am-org-role-mapping-service/values.preview.template.yaml
Comment (mattnayler) [inline charts/am-org-role-mapping-service/values.preview.template.yaml:38]: NB: This temporary change must be reverted before merging.

**Review by mattnayler — APPROVED** [review]
Re-approved after 58761c62b13aa77c245d529cb6d5e32c0fd33165.

File: charts/am-org-role-mapping-service/values.preview.template.yaml
Comment (mattnayler) [reply charts/am-org-role-mapping-service/values.preview.template.yaml]: Resolved in 58761c62b13aa77c245d529cb6d5e32c0fd33165.

**Pull request merged**
